templates/visuals/rvisual/src/visual.ts

```diff
@@ -73,8 +79,44 @@ module powerbi.extensibility.visual {
             this.imageDiv.style.width = finalViewport.width + 'px';
         }
 
+        /**
+         * This function gets called from the update function above. You should read the new values of the properties into 
+         * your settings object so you will be able to use the new value in the enumerateObjectInstances function below.
+         * 
+         * Below there is a code snippet for a case where you want to expose a single property called "lineColor" from the object called "settings"
+         * This object and property should be first defined in the capabilities.json file in the objects section.
+         * In this code we get the property value from the objects (and have a default value in case the )
+         */
+        public updateObjects(objects: DataViewObjects) {
+            /*this.settings = <VisualSettings>{
+                lineColor: getFillValue(object, 'settings', 'lineColor', "#333333")
+            };*/
+        }
+
+        /** 
+         * This function gets called on each of the objects defined in the capabilities files and allow you to select which of the 
```

Comment:
This function gets called **_for_** each of the objects defined in the capabilities files and **_allows_** you to select which of the